.env.template

```diff
@@ -13,6 +13,12 @@
 ## AI_SETTINGS_FILE - Specifies which AI Settings file to use (defaults to ai_settings.yaml)
 # AI_SETTINGS_FILE=ai_settings.yaml
 
+## OPENAI_API_BASE_URL - Custom url for the OpenAI API, useful for connecting to custom backends. No effect if USE_AZURE is true, leave blank to keep the default url 
+# OPENAI_API_BASE_URL=
+
+## EMBED_DIM - Define the embedding vector size, useful for models. OpenAI: 1536 (default), LLaMA 7B: 4096, LLaMA 13B: 5120, LLaMA 33B: 6656, LLaMA 65B: 8192 (Default: 1536)
+# EMBED_DIM=1536
```

Comment:
This seems fraught for mistakes is there anywhere these can be pulled from in the codebase?

Reply:
As you can see from the changes, EMBED_DIM was hardcoded as 1536 in every case it was used before.
I'm leaving 1536 as default and letting the user use whatever he needs.
Those example have been found by Kendel (https://github.com/keldenl/gpt-llama.cpp/issues/2#issuecomment-1514353829), they are from here
https://huggingface.co/shalomma/llama-7b-embeddings#quantitative-analysis

Anyway, it's just an example. If no one touches that value, it remains 1536 just as before